hs_core/views/resource_folder_hierarchy.py

```diff
@@ -73,7 +74,7 @@ def data_store_structure(request):
         for dname in store[0]:  # directories
             d_pk = dname.decode('utf-8')
             name_with_full_path = os.path.join(res_coll, d_pk)
-            d_url = istorage.url(os.path.join('zips', name_with_full_path))
+            d_url = to_external_url(istorage.url(os.path.join(name_with_full_path)))
```

Comment:
@sblack-usu I believe you missed ```zips``` in your ```os.path.join()``` call.

Reply:
zips isn't part of the external url.  The external url will redirect to the url that I deleted here.  See line 491 in hs_core/views/__init__.py

Reply:
Then you don't need ```os.path.join()``` as there is only one parameter ```name_with_full_path```, right?

Reply:
correct, I'll remove that.